app/Http/Controllers/CampaignsController.php

```diff
@@ -91,7 +91,11 @@ public function showInbox($campaignId)
      */
     public function showCampaign($id)
     {
-        $signups = Signup::campaign([$id])->has('posts')->with('posts')->orderBy('created_at', 'desc')->paginate(50);
+        // @TODO: currently we do not allow filtering so we only show accepted posts, so
+        // remove the contraint on status once filtering is rockin' 'n' rollin' again
+        $signups = Signup::campaign([$id])->has('posts')->with(['posts' => function ($query) {
+            $query->where('status', '=', 'accepted');
```

Comment:
@katiecrane yes I think we will! So on page load, we'll only show accepted posts and then when we filter, we'll send an API request to `GET /posts` that @sbsmith86 is building out now to filter by those statuses etc. to return the posts we want to display to help with performance on initial page load.